### DISCUSSION TITLE
Welcome to remoteapptool Discussions!

### DISCUSSION THREAD

**Discussion by kimmknight** [kimmknight/remoteapptool#104] [Announcements]
Hello RemoteApp Tool users,

Welcome to the GitHub Discussions section for RemoteApp Tool 😀

🌟 Thank You!
Thank you for your support, feedback, and contributions.

🤔 Questions
Feel free to post your questions, thoughts and discoveries related to RemoteApp Tool or RemoteApps in general.

💬 Issue Resolution
If you encounter issues, consider posting them here first before heading to the Issues section.

🚀 Feedback and Suggestions
Share your ideas and suggestions for RemoteApp Tool.

🔄 Transition from Google Group
GitHub Discussions for RemoteApp Tool will replace the Google Group called RemoteApp Tool.

Happy RemoteApp-ing!

**Comment by ustix** [comment]
Hello!

My question is: How legal is this approach of the RemoteApp usage? I mean, in normal circumstances you will need an RDP license server, and RDS User cals to even start up a RemoteApp, but this way you can just make it without it. How it is even possible?

**Reply by kimmknight** [reply]
My understanding is:

With Windows clients (except Home edition) and Server, you are generally permitted to host a single standard full-desktop RDP session for administrative purposes.

When using RemoteApps, you will be confined to the same restrictions. This is, you only need additional licencing if you wish to utilise more than one concurrent session, and/or, you wish to use it for non-administrative purposes.

Disclaimer: I am not a Microsoft licencing expert (or even a novice), so check with Microsoft before deploying if you are unsure. You are responsible for your own compliance.

I welcome input from anyone with more experience with Microsoft RDP licencing!

**Reply by MrBrianGale** [reply]
I too am not a Microsoft RDP licensing expert, but I've worked with them on some things before. My experience with working with Microsoft in terms of licensing around Desktop Windows is that you can RDP to the machine with 2 active connections, but one is on the admin console (which should be used sparingly) and one to the normal workstation. If you try to do more, then it will disconnect or deny the connection. Server edition, from my understanding, allows 1 connection to the admin console so you can do the initial configuration, but if you want more than 1 connection, you need to have licenses (CALS).

With desktop editions of Windows, you can have 1 user connected to the machine at any given time. This includes remote apps. The exception is with the admin connection (also called the console connection). So if you try to run 2 remote app sessions, the first one will disconnect when the second one starts (configuration dependent... it can reject the second connection instead of disconnecting the first).
For server side of things, you still need RDS User Cals to use remote apps, same as you would with normal setup. There is NO license circumvention happening with this application.

To reiterate, this app follows and requires things to follow Microsoft licensing for remote apps. You MUST have a User CAL to use the tool on Server and on desktop you can only have 1 active session at a time.

**Reply by ustix** [reply]
Thank you both for the answers, I thought the same!
